specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/2021-04-30-preview/iotcentral.json

```diff
@@ -1,23 +1,22 @@
 {
   "swagger": "2.0",
   "info": {
-    "version": "preview",
     "title": "Azure IoT Central",
-    "description": "Azure IoT Central is a service that makes it easy to connect, monitor, and manage your IoT devices at scale."
+    "description": "Azure IoT Central is a service that makes it easy to connect, monitor, and manage your IoT devices at scale.",
+    "version": "preview"
```

Comment:
preview version?

Reply:
Yeah, previously we used ever green preview version in our API, this is need to to ensure the backward compatibility, newer version of preview will had a date based versioning.